raiden/network/proxies/payment_channel.py

```diff
@@ -217,7 +208,7 @@ def update_transfer(
 
     def unlock(
             self,
-            merkle_tree_leaves: MerkleTreeLeaves,
+            merkle_tree_leaves: Optional[MerkleTreeLeaves],
```

Comment:
Updated the signature to match the token network. Even though I think the merkle tree should not be optional (a `None` value is an error)